src/SwiftTwigMailTemplate.php

```diff
@@ -116,7 +129,11 @@ public function renderMail(array $data = []) :\Swift_Message
 
         $twigMailExtension->pushMessage($mail);
 
-        $template = $this->twigEnvironment->loadTemplate($this->twigPath);
+        if (!empty($this->twigPath)) {
+            $template = $this->twigEnvironment->loadTemplate($this->twigPath);
+        } else if (!empty($this->twigContent)) {
+            $template = $this->twigEnvironment->createTemplate($this->twigContent);
+        }
```

Comment:
There will be a bug here if the user does provide neither $twigPath NOR $twigContent.
You should add a "else" and throw an exception.